tests/unittests/algo/test_tpe.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for :mod:`orion.algo.tpe`."""
+
+import numpy
+import pytest
+
+from orion.algo.space import Integer, Real, Space
+from orion.algo.tpe import adaptive_parzen_estimator, compute_max_ei_point, GMMSampler, TPE
+
+
+@pytest.fixture()
+def space():
+    """Return an optimization space"""
+    space = Space()
+    dim1 = Real('yolo1', 'uniform', -3, 6)
+    space.register(dim1)
+    dim2 = Real('yolo2', 'uniform', -2, 4)
+    space.register(dim2)
+
+    return space
+
+
+@pytest.fixture
+def tpe(space):
+    """Return an instance of TPE."""
+    return TPE(space)
+
+
+@pytest.fixture
+def gmm_sampler(tpe):
+    """Return an instance of GMMSampler"""
+    mus = numpy.linspace(-3, 3, num=12, endpoint=False)
+    sigmas = [0.5] * 12
+
+    return GMMSampler(tpe, mus, sigmas, -3, 3)
+
+
+def test_compute_max_ei_point():
+    """Test that max ei point is computed correctly"""
```

Comment:
Code is fine, but test would be more _severe_ if you would shuffle the data to test, otherwise answer is just last item and a buggy function could easily slip through unnoticed. But again, `compute_max_ei_point` is fine. :)